documentation/src/docs/asciidoc/user-guide/extensions.adoc

```diff
@@ -241,6 +241,21 @@ Examples:
 - `org.example.MyCondition`: deactivates the condition whose FQCN is exactly
   `org.example.MyCondition`.
 
+[[extensions-test-instance-factories]]
+=== Test Instance Factories
+`{TestInstanceFactory}` defines the API for `Extensions` that wish to _provide new_
```

Comment:
A corresponding link must also be added to `/documentation/src/docs/asciidoc/link-attributes.adoc`.

Reply:
👍